config.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   defaultArgs: {
     uri: '/document-download/:documentName',
-    documentNamePath: 'value.DocumentFileName'
+    documentNamePath: 'value.DocumentFileName',
+    documentWhiteList: []
```

Comment:
Just a minor nitpick, `documentWhitelist` is valid too

Reply:
what do you mean?

Reply:
I think I'm used to `whitelist` as one word so no need to camelcase the L there but I'm not 100% sure which is more accurate so happy with it either way 👍